lib/ui/screen-components/un-protected/layout/app-bar/index.tsx

```
@@ -260,10 +260,25 @@ const AppTopbar = ({ handleModalToggle }: IAppBarProps) => {
     );
   };
 
+  function fittedAddress(address:String | undefined)
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)